src/applications/vaos/actions/appointments.js

```diff
@@ -217,6 +238,71 @@ export function fetchPastAppointments(startDate, endDate, selectedIndex) {
   };
 }
 
+export function fetchExpressCareWindows() {
+  return async (dispatch, getState) => {
+    dispatch({
+      type: FETCH_EXPRESS_CARE_WINDOWS,
+    });
+
+    const initialState = getState();
+    const appointments = initialState.appointments;
+    let parentFacilities = appointments.parentFacilities;
+    const userSiteIds = selectSystemIds(initialState);
+
+    try {
+      if (!parentFacilities) {
+        parentFacilities = await getOrganizations({
+          siteIds: userSiteIds,
+          useVSP: false,
```

Comment:
When I was testing this with VSP set to true, the IDs I was getting back further down in didn't seem to be matching up with our `.json`.  If this is unexpected, I can go back and figure out why within this PR.